src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/EmptyBlockSectionProcessor.scala

```diff
@@ -0,0 +1,20 @@
+package org.ergoplatform.nodeView.history.storage.modifierprocessors
+
+import org.ergoplatform.modifiers.{BlockSection, ErgoPersistentModifier}
+import scorex.core.consensus.History.ProgressInfo
+
+import scala.util.{Failure, Try}
+
+/**
+  * Trait, that implements BlockSectionProcessor interfaces for regime, where node only
```

Comment:
Trait that implements BlockSectionProcessor interfaces for a regime where the node only downloads block headers